src/state/settings/settings.actions.ts

```diff
@@ -0,0 +1,21 @@
+import actionCreatorFactory from 'typescript-fsa'
+
+const action = actionCreatorFactory('SETTINGS')
+
+export interface AutorunSettingsParams {
+  autorunSimulation: boolean
+}
+
+export interface LogScaleSettingsParams {
+  logScale: boolean
+}
+
+export interface ShowHumanizedSettingsParams {
+  showHumanized: boolean
+}
+
+export const setAutorunSimulation = action<AutorunSettingsParams>('SETTINGS_SET_AUTORUN_SIMULATION')
```

Comment:
There are a few other settings that are currently persisted to LocalStorage, `SUPPRESS_DISCLAIMER` and `SKIP_LANDING_PAGE`.  You may want to include these for consistency.

Reply:
> There are a few other settings that are currently persisted to LocalStorage, `SUPPRESS_DISCLAIMER` and `SKIP_LANDING_PAGE`. You may want to include these for consistency.

SKIP_LANDING_PAGE is part of the ui settings reducer. Do you think it would be a good idea to combine SKIP_LANDING_PAGE and SUPPRESS_DISCLAIMER into a single reducer. If so, should I combine them into the ui reducer or the settings reducer. I would prefer settings reducer IMO.